pkg/neonvm/controllers/vm_controller_cpu_scaling.go

```diff
@@ -0,0 +1,119 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	corev1 "k8s.io/api/core/v1"
+
+	vmv1 "github.com/neondatabase/autoscaling/neonvm/apis/neonvm/v1"
+	"github.com/neondatabase/autoscaling/pkg/api"
+)
+
+// handleCPUScaling is extracted from doReconcile and encapsulates the logic to handle CPU scaling.
```

Comment:
suggestion:

> handleCPUScaling is extracted from doReconcile

IIUC, that's not relevant to what this function is / does, right? that's just the history?

If so, I'd just remove that part, so it reads

> handleCPUScaling encapsulates the logic to handle CPU scaling.

Reply:
rephrased comment